kview/src/main/java/dev/ikm/komet/kview/mvvm/viewmodel/PatternViewModel.java

```diff
@@ -216,7 +219,36 @@ public boolean createPattern() {
                                     .acceptability(ACCEPTABLE))
             );
         }
+        boolean isSuccess = composer.commitSession(session);
+        setPropertyValue(PATTERN, pattern);
+        updateStamp();
+        return isSuccess;
+    }
+
+    public String getPatternTitle() {
+        return ((EntityFacade) getPropertyValue(PATTERN)).description();
+    }
+
+    public ViewProperties getViewProperties() {
+        return getPropertyValue(VIEW_PROPERTIES);
+    }
+
+    public void updateStamp() {
+        EntityFacade patternFacade = getPropertyValue(PATTERN);
+        StampCalculator stampCalculator = getViewProperties().calculator().stampCalculator();
+
+        StampViewModel stampViewModel = getPropertyValue(STAMP_VIEW_MODEL);
+
+        Stamp stamp = stampCalculator.latest(patternFacade).get().stamp();
+        stampViewModel.setValue(STATUS, stamp.state());
+        stampViewModel.setValue(TIME, stamp.time());
+        stampViewModel.setValue(AUTHOR, stamp.author());
+        stampViewModel.setValue(MODULE, stamp.module());
+        stampViewModel.setValue(PATH, stamp.path());
+    }
 
-        return composer.commitSession(session);
+    public String getPatternIdentifierText() {
+        EntityFacade patternFacade = getPropertyValue(PATTERN);
+        return String.valueOf(patternFacade.toProxy().publicId().asUuidList().getLastOptional().get());
```

Comment:
Can we look at how our recent change from the concept window? 
It has code to also display the SNOMED and LOINC codes if present.